LibreNMS/Plugins/DeviceHook.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace LibreNMS\Plugins;
+
+use App\Models\Device;
+
+trait DeviceHook
+{
+    public static function device_overview_container($device)
```

Comment:
All methods should be camel case.

Reply:
This one is the historical one for plugins. Meaning changing it will be a challenge for backward compatibility and would require users to patch existing plugins, and notification will be necessary.